sdk/identity/identity/review/identity.api.md

```diff
@@ -42,14 +43,15 @@ export class AzureCliCredential implements TokenCredential {
     constructor();
     protected getAzureCliAccessToken(resource: string): Promise<unknown>;
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
-}
+    }
 
 // @public
 export type BrowserLoginStyle = "redirect" | "popup";
 
 // @public
 export class ChainedTokenCredential implements TokenCredential {
     constructor(...sources: TokenCredential[]);
+    protected credentialName: string;
```

Comment:
This is so that the ChainedTokenCredential can have a custom name.

Reply:
I don't know about this. A chained token credential would be an abstraction away from any one credential.

Reply:
I'm mainly wondering about how to send in the name of the DefaultAzureCredential, so that the logs render properly.

Reply:
I'm not sure what else I can do for that use case. Suggestions appreciated!